include/odp/api/abi-default/ipsec.h

```diff
@@ -29,6 +29,7 @@ typedef struct { char dummy; /**< @internal Dummy */ } _odp_abi_ipsec_sa_t;
 typedef _odp_abi_ipsec_sa_t *odp_ipsec_sa_t;
 
 #define ODP_IPSEC_SA_INVALID ((odp_ipsec_sa_t)0)
+#define ODP_IPSEC_SA_INVALID_COMBO ((odp_ipsec_sa_t)-1)
```

Comment:
This is an ODP API change that needs to be done also in the API spec headers, not only in the ABI files. And the commit title should be prefixed with "api: crypto:" for such changes.